tests/cephfs/cephfs_utilsV1.py

```diff
@@ -1455,6 +1455,18 @@ def create_fs(self, client, vol_name, validate=True, **kwargs):
             volname_ls = json.loads(out)
             if vol_name not in [i["name"] for i in volname_ls]:
                 raise CommandFailed(f"Creation of filesystem: {vol_name} failed")
+
+        log.info("Validating the creation of pools")
+        outpools, rcpools = client.exec_command(
+            sudo=True,
+            cmd=f"ceph osd lspools | grep -E 'cephfs.{vol_name}.data|cephfs.{vol_name}.meta' | wc -l",
```

Comment:
can you use -f json will be easy to parse

```
ceph osd lspools -f json

[{"poolnum":1,"poolname":"cephfs.cephfs.meta"},{"poolnum":2,"poolname":"cephfs.cephfs.data"},{"poolnum":3,"poolname":".mgr"},{"poolnum":4,"poolname":"cephfs-ec-data-ec"},{"poolnum":5,"poolname":"cephfs-ec-metadata-ec"},{"poolnum":6,"poolname":".nfs"}]

```

Reply:
@AmarnatReddy Had the same discussion with Hemanth. After getting output with JSON, it needs to be again validated using the similar set of query. To reduce the code line and make validation simple, used this logic